pkg/simple/client/servicemesh/options.go

```diff
@@ -20,9 +20,6 @@ import "github.com/spf13/pflag"
 
 type Options struct {
 
-	// istio pilot discovery service url
```

Comment:
We really shouldn't remove this field this quick. I think it's better to deprecate it first, then remove it in the following releases. In this way, we could run KubeSphere with the old version configuration file, which will not cause inconsistency.

Reply:
ok

Reply:
any update?